rest_tools/utils/auth.py

```diff
@@ -28,6 +29,11 @@ def _validate(self, token, key, **kwargs):
         claims.extend(kwargs.pop('required', []))
         options['require'] = claims
 
+        leeway = kwargs.pop('leeway', None)
+        if leeway is None:
+            leeway = self.leeway
+        leeway = float(leeway)
+
```

Comment:
In practice, I don't understand the reasoning behind handling both `self.leeway` and a param `leeway`. I presume that multiple unique entrypoints lead to `_validate()` and one of those does not set `_AuthValidate.validate` (or really, one of it's derived classes). However it does fit the established pattern, so this is fine.